site/content/internal/infrastructure/grafana.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Community Grafana"
+date: 2020-04-01T16:08:19+02:00
+subsection: internal
+weight: 20
+---
+
+## Community Grafana
+
+The Grafana application is used to visualise the performance metrics of the community installations. The application runs in the Mattermost Cloud infrastructure in the same cluster that community installations are hosted.
+
+Grafana URL: https://grafana.mattermost.com
+
+After the migration to the cloud infrastructure the community installations are created by the cloud provisioner. Therefore, the naming has changed and another type of Prometheus label is required to select which installation you want to get data for.
```

Comment:
I would exclude talking about the migration or how it was in the past. That will just confuse, or adds unnecessary information, the people who read this doc in the future who don't know anything about the migration

Reply:
Thanks fixed